helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -178,7 +178,7 @@ private void joinCluster() {
           LOG.info(_instanceName + " is auto-registering cluster: " + _clusterName);
           CloudInstanceInformation cloudInstanceInformation = getCloudInstanceInformation();
           String domain = cloudInstanceInformation
-              .get(CloudInstanceInformation.CloudInstanceField.FAULT_DOMAIN.name());
+              .get(CloudInstanceInformation.CloudInstanceField.FAULT_DOMAIN.name()) + _instanceName;
```

Comment:
This is domain string right? Then it should looked like "FD=xxx, InstanceName=xxxx". But the code is adding them together?

Reply:
The prefix is in the processor: 
//The hostname will be filled in by each participant
 String domain = parts[0] + "=" + platformFaultDomain + "," + parts[1] + "=";